pyodide_build/buildpkg.py

```diff
@@ -412,6 +375,105 @@ def _compile(
                 self.src_extract_dir, self.src_dist_dir, build_env, config_settings
             )
 
+    def _patch(self) -> None:
```

Comment:
There is no change in `_patch` or other functions. It seems git diff is not smart enough to create a small diff.